src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,108 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    static final int DEFAULT_INITIAL_CAPACITY = 1 << 4;
+    static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    static final int RESIZE_FACTOR = 2;
+    private Node<K,V>[] table;
+    private int threshold;
```

Comment:
the value of field is read only once, and updated multiple times, it's not worth keeping as a field